.gitbook/assets/_api-reference-docs-openapi.json

```diff
@@ -503,6 +503,13 @@
               "maximum": 100,
               "minimum": 1
             }
+          },
+          {
+            "name": "sortBy",
+            "in": "query",
+            "description": "Sort results by a field. Accepted fields are: `roomName`, `startDate` and `sizeInMegaBytes`. Accepted sort values are 'asc' and 'desc'.",
```

Comment:
Maybe instead of "values" we could use direction, I think I've seen it in other places.

Reply:
That's probably a larger conversation/refactor of how the API endpoint actually functions though, correct?

Reply:
No, we are just talking about how we refer to the `direction` prop here. No refactor needed. We can use direction instead of value, but this is also fine 👍